### PR TITLE
add article number

### PR DESCRIPTION
**Pull request by pdebuyl**
See https://github.com/ReScience/ReScience/issues/47 and https://github.com/ReScience/ReScience-submission/pull/29

### CODE REVIEW

File: 03-edit.md [hunk 1]
```diff
@@ -139,6 +139,8 @@ that are listed here:
     - Notebook repository (if necessary)
     - Data repository (if necessary)
     - Volume, issue and year
+    - Article number (**only** use the number provided by one of the
+      Editors-in-Chief in the submission issue).
```
Comment (rougier) [inline 03-edit.md]: Do we need editorin-chief for that ? Maybe we can have a running issue where editors tell they'll use this number for this article, no ?

Comment (pdebuyl) [inline 03-edit.md]: Fair point :-) At https://github.com/ReScience/ReScience ?

Comment (rougier) [inline 03-edit.md]: Yes.

Comment (pdebuyl) [inline 03-edit.md]: Done https://github.com/ReScience/ReScience/issues/48

**Pull request merged**